### PR TITLE
New package: drivestrike-2.1.20

### PR DESCRIPTION
**Pull request by jcgruenhage**
Not sure if this is interesting for anyone here, but we have to deploy this onto our machines for work, so I created this template to install it on my work laptop. EULA says no copies except for subscribers installing it onto their systems, so `restricted` it is.

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

**Comment by madonius** [comment]
I would propose updating the Package to 2.1.21-2.
Moreover, I would recommend extracting the information from the [Packages](https://app.drivestrike.com/static/apt/dists/stretch/main/binary-amd64/Packages) file

**Comment by jcgruenhage** [comment]
Automatically extracting that for the sake of the template doesn't make much sense, but using the Packages file for the `update` file to fetch what the most recent version is would definitely make sense. Thanks for the hint :)

File: srcpkgs/drivestrike/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+license="custom:Proprietary"
+homepage="https://drivestrike.com"
+distfiles="https://app.drivestrike.com/static/apt/pool/main/d/drivestrike/${_filename}"
+checksum=8dc4c1b85ea316b71adc08e7807b03b609b81a1f7f69c086245fce83461f3515
+repository=nonfree
+restricted=yes
+nostrip=yes
+
+do_build() {
+	ar x "${_filename}"
```
Comment (Piraty) [inline srcpkgs/drivestrike/template:20]: look at c18090b458b704ca314d76c4e198a546443ae168

Comment (jcgruenhage) [reply srcpkgs/drivestrike/template:20]: lgtm, will adapt. Thanks!

File: srcpkgs/drivestrike/files/drivestrike/log/run [hunk 2]
```diff
@@ -0,0 +1 @@
+/usr/bin/vlogger
```
Comment (Piraty) [inline srcpkgs/drivestrike/files/drivestrike/log/run:1]: see https://github.com/void-linux/void-packages/pull/42026

**Review by Piraty — CHANGES_REQUESTED** [review]

**Pull request merged**